src/Areas/AppConfig/Commands/KeyValue/KeyValueUnlockCommand.cs

```diff
@@ -53,7 +56,7 @@ await appConfigService.UnlockKeyValue(
         catch (Exception ex)
         {
             _logger.LogError(ex, "An exception occurred unlocking key. Key: {Key}.", options.Key);
-            HandleException(context.Response, ex);
+            HandleException(context, ex);
```

Comment:
Can the _logger.LogError call be inside of HandleException?

Reply:
We had an issue about this filed.

Reply:
Can you file a refactor issue for this?

Reply:
https://github.com/Azure/azure-mcp/issues/514